migrations/migrations.sql

```diff
@@ -809,6 +809,15 @@ do $$
       alter table "resources" add column "branch" varchar(255);
     end if;
 
+    -- Reindex resourceSubscriptionIdNameU to resourceSubscriptionIdNameBranchU in resources table
+    if exists (select 1 from pg_indexes where tablename = 'resources' and indexname = 'resourceSubscriptionIdNameU') then
+      drop index "resourceSubscriptionIdNameU";
+    end if;
+
+    if not exists (select 1 from pg_indexes where tablename = 'resources' and indexname = 'resourceSubscriptionIdNameBranchU') then
```

Comment:
I think lets switch the order of the migrations. At least one of these constraints will exist at all times then. Also, I think we should add this in the launch tasks so the column/index are created before the deployment and does not delay it.